tools/test.py

```diff
@@ -196,7 +196,7 @@ def Done(self):
         print failed.output.stdout.strip()
       print "Command: %s" % EscapeCommand(failed.command)
       if failed.HasCrashed():
-        print "--- CRASHED ---"
+        print "--- CRASHED (%d) ---" % -failed.output.exit_code
```

Comment:
Perhaps it might be helpful to add a prefix before the number here and below, like "Exit code ".